roles/jboss_fuse_on_karaf/tasks/main.yml

```diff
@@ -64,15 +64,15 @@
 # Look for fuse inside KARAF_HOME directories
 
 - name: check bin/fuse
-  raw: ls -1 "{{ item }}/bin/fuse"
+  raw: ls -1 "{{ item }}"/bin/fuse 2>/dev/null
   register: karaf_home_bin_fuse
   ignore_errors: yes
   with_items: "{{ karaf_homes }}"
   when: 'jboss_fuse'
 
 
 - name: check system/org/jboss/fuse
-  raw: ls -1 "{{ item }}/system/org/jboss"
+  raw: ls -1 "{{ item }}"/system/org/jboss 2>/dev/null
```

Comment:
I see on most of these you moved the quotes to just be around `"{{item}"`. Do they disappear after the templating has done its job?

Reply:
yes the quotes disappear.